OWNER_ALIASES: Delete all users from Google, add QE + Engineering

mvinkler opened the pull request:
This PR modifies OWNER_ALIASES: deleted all users from Google, added QE (mvinkler, mgencur, afield) + Engineering (bbrowning, bobmcwhirter, jcrossley3, Justin Lee, markusthoemmes, Vincent Demeester). More people may be added later.

mgencur commented:
Hmm, I'd say we need to modify OWNERS files as well. The owners file can include either Github names or aliases defined in OWNERS_ALIASES. But I think changing just OWNERS_ALIASES is not enough. Prow will look at the OWNERS file.

mgencur commented:
Hold on. We might be able to generate these files in CI. I'm looking into how to do it.

mgencur commented:
OK. I think it's not possible to generate the files outside the target repos. And still think the OWNERS file has to be changed too.

mvinkler commented:
Introduced new aliases "serving-approvers" + "serving-reviewers", added these aliases into the OWNERS file in the parent folder. This way, all other aliases become redundant and the list of users in these aliases can be empty (as each alias group has the same users). @mgencur WDYT?

mgencur commented:
LGTM. Merging.
